docs/supported_apis/dataframe_supported.rst

```diff
@@ -6,641 +6,34 @@ of an operation that is listed as not implemented, feel free to open an issue on
-.. _`transpose`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.transpose.html#pandas.DataFrame.transpose
-.. _`truediv`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.truediv.html#pandas.DataFrame.truediv
-.. _`truncate`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.truncate.html#pandas.DataFrame.truncate
-.. _`tshift`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.tshift.html#pandas.DataFrame.tshift
-.. _`tz_convert`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.tz_convert.html#pandas.DataFrame.tz_convert
-.. _`tz_localize`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.tz_localize.html#pandas.DataFrame.tz_localize
-.. _`unstack`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.unstack.html#pandas.DataFrame.unstack
-.. _`update`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.update.html#pandas.DataFrame.update
-.. _`value_counts`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.value_counts.html#pandas.DataFrame.value_counts
-.. _`values`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.values.html#pandas.DataFrame.values
-.. _`var`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.var.html#pandas.DataFrame.var
-.. _`where`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.where.html#pandas.DataFrame.where
-.. _`xs`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.xs.html#pandas.DataFrame.xs
+.. table::
+   :widths: 1, 5
+
+   +----------+-----------------------------------------------------------------------------------------------+
+   | Flag     | Meaning                                                                                       |
+   +==========+===============================================================================================+
+   | Harmful  | Usage of this parameter can be harmfull for performance of your application                   |
+   +----------+-----------------------------------------------------------------------------------------------+
+   | Non-lazy | Usage of this parameter can trigger non-lazy execution (actual for OmniSci execution only)    |
+   +----------+-----------------------------------------------------------------------------------------------+
+   | Partial  | Parameter can be partly unsupported, it's usage can be harmfull for performance of your       |
+   |          | appcication                                                                                   |
+   +----------+-----------------------------------------------------------------------------------------------+
+   | pandas   | Usage of this parameter, triggers usage of original pandas function, no performance           |
```

Comment:
This flag intended to the cases when we use pandas functions without any modification, like `eval` function. This function is just imported from pandas in `modin/pandas/__init__.py`, check:
`python -c "import modin.pandas as pd; import pandas; print(pd.eval is pandas.eval)"` # gives True
In this case `d2p` stands for something different in terms of Modin (combining partitions, function applying, and data splitting data back across partitions)

Reply:
Oh, I see. Left a small suggestion.

Reply:
However, we shouldn't use these functions with Modin objects because pandas doesn't know nothing about Modin.

Reply:
Is there an example for using `pandas` flag in this PR?

Reply:
> However, we shouldn't use these functions with Modin objects because pandas doesn't know nothing about Modin.

Most of the imported methods don't use any pandas or Modin objects, so it should be fine to use them. In the same time there are some methods can use such objects (for example [pandas.qcut](https://pandas.pydata.org/docs/reference/api/pandas.qcut.html)), so, i think, these functions should be checked (maybe we should add some wrappers for checking Modin objects).

> Is there an example for using pandas flag in this PR?

Examples can be found in the [built docs](https://modin--4286.org.readthedocs.build/en/4286/supported_apis/utilities_supported.html). We are [handling](https://github.com/modin-project/modin/pull/4286/files#diff-389286cc6854493992e37207836ebbb832d8e5337141a099ca800ab749f7b5c4R263-R266) these methods in the `scripts/supported_apis.py` script.

Reply:
What do you think on replacing `pandas` flag to `pure pandas`?

Reply:
For me this option is fine, implemented in the last commit.